### DISCUSSION TITLE
2023.8.3

### DISCUSSION THREAD

**Discussion by AndrewBastin** [hoppscotch/hoppscotch#3511] [Announcements]
This release introduces some new features pertaining to the launch of the Hoppscotch Desktop App, most notably support for cookies.

## What's Changed
* HBE-270 Test-Case timestamp issue fix in backend by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/3415
* HBE-266 Update NestJS packages by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/3389
* refactor: removed all instances of `rejectOnNotFound` in prisma queries by @balub in https://github.com/hoppscotch/hoppscotch/pull/3377
* refactor: improvements to the auth implementation in admin dashboard by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/3444
* fix(self-hosted/web): add "useCredentials: true" to Vite PWA options by @superbiche in https://github.com/hoppscotch/hoppscotch/pull/3460
* refactor: move sentry to platform by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/3451
* fix(common): :bug: allows URL parsing as obsvered in #1873 by @kpguru20001 in https://github.com/hoppscotch/hoppscotch/pull/3465
* refactor: versioning and migration mechanism for public data structures by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/3457
* fix: graphql page crashing and broken syntax highlighting  by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/3488
* chore(common): implement enforced pre-commit type checks for FE service files by @jamesgeorge007 in https://github.com/hoppscotch/hoppscotch/pull/3472
* chore(data): bump dependencies by @jamesgeorge007 in https://github.com/hoppscotch/hoppscotch/pull/3473
* fix: offset by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3494
* chore: fix spelling mistake on type import by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3487
* fix: add optional chaining for teamMembers to handle undefined team by @nmerget in https://github.com/hoppscotch/hoppscotch/pull/3484
* fix: mailer template issue by @balub in https://github.com/hoppscotch/hoppscotch/pull/3475
* fix: graphql query deprecation issue by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3506
* chore: updated brand assets by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3500

## New Contributors
* @superbiche made their first contribution in https://github.com/hoppscotch/hoppscotch/pull/3460
* @kpguru20001 made their first contribution in https://github.com/hoppscotch/hoppscotch/pull/3465
* @nmerget made their first contribution in https://github.com/hoppscotch/hoppscotch/pull/3484

**Full Changelog**: https://github.com/hoppscotch/hoppscotch/compare/2023.8.2...2023.8.3

<hr /><em>This discussion was created from the release <a href='https://github.com/hoppscotch/hoppscotch/releases/tag/2023.8.3'>2023.8.3</a>.</em>

**Comment by Igorgro** [comment]
Still cannot self-host hoppscotch due to this bug #3378 😢
